__tests__/app.test.js

```diff
@@ -337,3 +337,29 @@ describe("DELETE /api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("/api/users", () => {
+  describe("GET", () => {
+    test("STATUS 200 - Responds with an array of all the user objects.", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then(({ body: { users } }) => {
+          expect(users).toHaveLength(4);
+          users.forEach(({ username, name, avatar_url }) => {
+            expect(typeof username).toBe("string");
+            expect(typeof name).toBe("string");
+            expect(typeof avatar_url).toBe("string");
+          });
+        });
+    });
+    test("STATUS 404 - Responds with 404", () => {
```

Comment:
(api/not-an-endpoint) This endpoint is an invalid end point so would not fall into the controller of the endpoint /api/users.
Therefore a test for an invalid end point such as this, would be in a separate describe block.
Something like /api/articles/monkey however would fall into the /api/articles/article_id end point, and would therefore be dealt with in the /api/articles/article_id test block. Hope that makes sense !
Make sure to move this 404 to it's own describe block